qa/rpc-tests/wallet_shieldcoinbase_ua_sapling.py

```diff
@@ -14,16 +14,16 @@ def test_init_zaddr(self, node):
         # this function may be called no more than once
         assert(self.account is None)
         self.account = node.z_getnewaccount()['account']
-        self.addr = node.z_getaddressforaccount(self.account)['unifiedaddress']
+        self.addr = node.z_getaddressforaccount(self.account)['address']
         return self.addr
 
     def test_check_balance_zaddr(self, node, expected):
         balances = node.z_getbalanceforaccount(self.account)
-        assert('transparent' not in balances['pools'])
-        assert('sprout' not in balances['pools'])
-        sapling_balance = balances['pools']['sapling']['valueZat']
+        assert('transparent' not in balances['receiver_types'])
+        assert('sprout' not in balances['receiver_types'])
+        sapling_balance = balances['receiver_types']['sapling']['valueZat']
```

Comment:
This assertion is failing in CI, due to a missing `receiver_types` key. That's due to what I presume is an over-zealous string replacement: `z_getbalanceforaccount` is an API that we _do_ want to return `pools` (and the RPC continues to do so).
```suggestion
        assert('transparent' not in balances['pools'])
        assert('sprout' not in balances['pools
        sapling_balance = balances['pools']['sapling']['valueZat']
```